numbers-booleans-objects/add-update-delete-objects/README.md

```diff
@@ -6,7 +6,16 @@ Here, we will learn to add, update and delete properties of an Javascript Object
 ### Adding New Properties
 There’re a few ways to add properties to an object in JavaScript. You can add new properties to an existing object by simply giving it a value.
 
-Assume that you've a *user* object that already exists. One way to add a property to this  *user* object is, by using the **dot notation**:
+Assume that you've an *user* object  with the following definition:
+```js
+const user = {
```

Comment:
@avishekjana The survey or the user form that we are building has 

- Name
- Email
- date of birth
- Password
- accept terms and conditions check box

So better not to have `firstName`, `lastName` to be consistent, right?

Reply:
OK